phpcs.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<ruleset name="Irving Example">
+	<description>PHP_CodeSniffer standard for Irving Example Theme.</description>
+
+	<!-- Include Alley Rules -->
+	<rule ref="alley-coding-standards">
+		<exclude-pattern>/tests/*</exclude-pattern>
+		<exclude-pattern>/inc/fields.php</exclude-pattern>
+	</rule>
+
+	<!-- Project customizations go here -->
+
+	<rule ref="WordPress">
+		<!-- Allow short ternary expressions -->
+		<exclude name="WordPress.PHP.DisallowShortTernary" />
+	</rule>
```

Comment:
I think we do this in the base Alley rules, so it may be unnecessary now. Not a blocker in any case.